### ISSUE TITLE
New endpoint for cavern app

### ISSUE THREAD

**Issue by secminhr** [t510599/cavern#18]
Help create an endpoint.
For every GET request, return a json
```json
{
 "author_key":"name",
 "author_username_key":"author",
 "comments_key":"comments",
 "comments_number_key":"comments_count",
 "content_key":"content",
 "date_key":"time",
 "email_key":"email",
 "hash_key":"hash",
 "id_key":"id",
 "is_liked":"islike",
 "likes_key":"likes",
 "login":"button.login.button",
 "login_key":"login",
 "markdown_key":"markdown",
 "message_key":"message",
 "names_key":"names", 
 "page_limit":"page_limit",
 "pid_key":"pid",
 "post_key":"post",
 "posts_count_key":"posts_count",
 "posts_key":"posts",
 "role_key":"level",
 "status_key":"status",
 "title_key":"title",
 "total_posts_num":"all_posts_count",
 "upvote_key":"likes_count",
 "username_key":"username"
}
```

**Comment by t510599** [comment]
I believed that this can be resolved by creating a static file instead of an 'endpoint'?
@a91082900 What do you think?

**Comment by secminhr** [comment]
This closes #19

**Comment by secminhr** [comment]
Thank you.
This closes #18
